### ISSUE TITLE
FinRL can learn and predict actions. but not price? At which price does an action take?

### ISSUE THREAD

**Issue by julianzero** [AI4Finance-Foundation/FinRL#919]
Hello everyone,

I found that FinRL can only learn which actions to take, i.e., buy or sell and quantity, via the neural networks during training.
What about price? Which price should each action takes at?
Is there a way to train and predict the bid or ask price with FinRL?

Thanx.

**Comment by notBradPitt** [comment]
> What about price? Which price should each action takes at?

It's described here: https://towardsdatascience.com/finrl-for-quantitative-finance-tutorial-for-multiple-stock-trading-7b00763b7530

I believe each action takes place after the market closes and hopefully you'll make a profit by the end of the next day.

**Comment by darenwai** [comment]
notBradPitt: Does this means that the agent only knows how to predict tomorrow price, therefore, only able to do day trading and not swing trading?

**Comment by notBradPitt** [comment]
I'm not too sure, but I think it's the opposite. The default timeframe is day, so you run the code once the market closes and the model determines what action should you take tomorrow. If the market closes at price 200 and your model says -2, it means you should put an order for a sell of 2 stocks at 200 when the market reopens.

I myself still need a clarification from the devs because I'm still confused with a lot if the aspects here

**Comment by darenwai** [comment]
you are right. so it isn't day trading. The model is capable of swing trading. If today 10, tomorrow 10, and next day 10 and next day -30. Means it is holding for 3days :)
